project/Dependencies.scala

```diff
@@ -27,6 +28,11 @@ object Dependencies {
     "com.typesafe.akka" %% "akka-stream" % versions.akka
   )
 
+  val akkaStreamDependencies = Seq(
+    "com.typesafe.akka" %% "akka-stream" % versions.akka,
```

Comment:
I think this should be removed from line 28 if it's moving here?

Reply:
Also `akkaStreamDependencies` should maybe something like ```akkaDependencies ++ Seq(...)``` since you can't have akka streams without akka

Reply:
Ah, I hadn’t noticed that was duplicated. And 👍 on making it `akkaDependencies ++`.